test/instrumentation/resque_test.rb

```diff
@@ -6,7 +6,9 @@
   require_relative "../jobs/resque/remote_call_worker_job"
   require_relative "../jobs/resque/error_worker_job"
 
-  Resque.redis = Redis.new(:url => 'redis://:secret_pass@localhost:6379')
+  Resque.redis = Redis.new(:host => ENV['REDIS_HOST'] || ENV['APPOPTICS_REDIS_SERVER'] || '127.0.0.1',
```

Comment:
This one had a different url and port than the other ones replaced  (`localhost:6379` rather than `127.0.0.1`). Should the default have been updated to `127.0.0.1`?

Reply:
good eye!
I'm trying to remember, but I think it was an issue with localhost not always mapping to 127.0.0.1, so using 127.0.0.1 is more reliable